### DISCUSSION TITLE
Internet is not working on Securityonion VM

### DISCUSSION THREAD

**Discussion by soniaazeem** [Security-Onion-Solutions/securityonion#7659] [Unsupported Versions]
Have just setup SO on VM ware workstation and got web access properly first time, but now internet is not working on security-onion VM consequently lost web access, While internet is working fine on other VMs and Host machine as well..
Since, I'm new to SO, I dont know what I'm missing exactly

**Comment by tomjones1977** [comment]
Does networking work at all?

How have you configured the VM network settings in VMWare?

**Comment by ghost** [comment]
I went into settings > network in the Security Onion VM, changed the "Wired" tab to "ON", and the internet started working again.
